Allow documentation for stored procedures (#1358)
mozilla/bigquery-etl: mozfun/event_analysis/README.md

# Event Analysis

These functions are specific for use with the `events_daily` and `event_types` tables.
By themselves, these two tables are nearly impossible to use since the event history
is compressed; however, these stored procedures should make the data accessible.

The `events_daily` table is created as a result of two steps:
1. Map each event to a single UTF8 char which will represent it
2. Group each client-day and store a string that records, using the
    compressed format, that clients' event history for that day.
    The characters are ordered by the timestamp which they appeared
    that day.

> We primarily enable access to this data by creating views which do the heavy lifting.

Reviewer: This slipped through before, but we suggest not using "we" in documentation:

https://docs.telemetry.mozilla.org/contributing/style_guide.html#writing-style

```suggestion
The best way to access this data is to create a view to do the heavy lifting.
```

> For example, to see which clients completed a certain action, we can create a view that

Reviewer: ```suggestion
For example, to see which clients completed a certain action, you can create a view using these functions that
```
